features/git-flow.feature

```diff
@@ -35,4 +36,20 @@ Feature: Git Flow
     Scenario: New Commit
         When a new commit is pushed
         And it is against the pipeline's branch
-        Then a new build from `main` should be created to test that change
+        Then a new build from "main" should be created to test that change
+
+    Scenario: New Tag
+        When a tag is created
+        Then a new build from "tag-triggered" should be created to test that change
+
+    Scenario: New Annotated Tag
+        When an annotated tag is created
+        Then a new build from "tag-triggered" should be created to test that change
+
+    Scenario: New Release
+        When a release is created
+        Then a new build from "release-triggered" should be created to test that change
+
+    Scenario: New Annotated Release
```

Comment:
How about `New Release with Annotated Tag`? There is no "annotated release" on GitHub

Reply:
I think so. I completely thought "release with annotated tag" was "annotated release".